backend/LeetWars.Core/LeetWars.Core.Common/DTO/Friendship/UpdateFriendshipDto.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using LeetWars.Core.DAL.Enums;
+using System.Text.Json.Serialization;
 
 namespace LeetWars.Core.Common.DTO.Friendship
 {
     public class UpdateFriendshipDto
     {
         public long UserId { get; set; }
         public long FriendshipId { get; set; }
+        [JsonConverter(typeof(JsonStringEnumConverter))]
```

Comment:
why do we need this?

Reply:
It is needed to work with the analogous enum on the frontend

Reply:
but we configured the same in out Program file

Reply:
Unfortunately it does not work without explicitly mentioning the converter, i guess something overrides it

Reply:
strange as it is works for all other places

Reply:
Found the reason, that was SignalR, now it has needed config